### PR TITLE
Docker compose improvements

### PR DESCRIPTION
**Pull request by shajidissa**
improvements to docker compose

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -55,3 +57,16 @@ services:
         retries: 6
     ports:
       - "1433:1433"
+
+  sql-data:
```
Comment (marrobi) [inline docker-compose.yml:61]: is `sql-data` the best name? or `sql-mts`, or `sql-mts-data` maybe?

File: docker-compose.yml [hunk 2]
```diff
@@ -55,3 +57,16 @@ services:
...
+    environment:
+      SA_PASSWORD: ${SAPASSWORD}
+      ACCEPT_EULA: "Y"
+    healthcheck:
+        test: ["CMD", "/opt/mssql-tools/bin/sqlcmd","-S sql-data,1434 -U sa -P ${SAPASSWORD} -Q 'SELECT * FROM INFORMATION_SCHEMA.TABLES'"]
+        interval: 10s
+        timeout: 10s
+        retries: 6
+    ports:
+      - "1434:1433"
```
Comment (marrobi) [inline docker-compose.yml:72]: I believe we only need to specify ports if we want to service to be exposed externally. I'd query if either SQL server need to be exposed externally. If not, try not exposing ports at all, they should both be accessible on 1433...

Doesn't really matter I guess as using compose for dev, but aware the port numbers are getting messier.

**Review by marrobi — COMMENTED** [review]
Not blockers, things that may want to think about.

File: docker-compose.yml (hunk 1 above)
Comment (shajidissa) [reply docker-compose.yml]: changed name to sql-mts-data

File: docker-compose.yml (hunk 2 above)
Comment (shajidissa) [reply docker-compose.yml:72]: QA are connecting to sql on their host machines. Useful for now.

Yeh, the ports will get messier once we get more micro services.

**Review by mdp0 — APPROVED** [review]

**Review by marrobi — APPROVED** [review]

**Pull request merged**